quartz.config.ts

```diff
@@ -48,19 +53,14 @@ const config: QuartzConfig = {
     transformers: [
       Plugin.FrontMatter(),
       Plugin.CreatedModifiedDate({
-        // you can add 'git' here for last modified from Git
-        // if you do rely on git for dates, ensure defaultDateType is 'modified'
         priority: ["frontmatter", "filesystem"],
       }),
       Plugin.Latex({ renderEngine: "katex" }),
       Plugin.SyntaxHighlighting({
-        // uses themes bundled with Shikiji, see https://shikiji.netlify.app/themes
         theme: {
           light: "github-light",
           dark: "github-dark",
         },
-        // set this to 'true' to use the background color of the Shikiji theme
-        // if set to 'false', will use Quartz theme colors for background
```

Comment:
let's keep all the inline notes here. Still be neat to have quick discussion here instead of having users to open the docs.

Docs should just be there for anyone who wants to read it. For people who just got here the first time, it might be overwhelming to redirect them to a configuration site.